<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWidget</class>
 <widget class="QWidget" name="MenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="balanceButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance Enquiry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transferButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfer Money</string>
   </property>
  </widget>
  <widget class="QPushButton" name="overflowButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Overflow Management</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw Money</string>
   </property>
  </widget>
  <widget class="QPushButton" name="periodicTrButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Periodic Transfer</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#message { font-size: 14pt; }</string>
   </property>
   <property name="text">
    <string>Select an option from the list below:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="supportButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Support</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
